public/templates/master.html

```
@@ -0,0 +1,189 @@
+            margin: 16px 0;
+        }
+
+        .footer-bottom {
+            padding: 0.5rem;
+            border-bottom-right-radius: 0.5rem;
+            border-bottom-left-radius: 0.5rem;
+            max-width: 37.5em;
+        }
+    </style>
+</head>
+<table align="center" width="100%" border="0" cellPadding="0" cellSpacing="0" role="presentation"
```

HTMLHint (1.5.0)

[error] The attribute name of [ cellPadding ] must be in lowercase.

(attr-lowercase)